Esterni/VerbaliEsterni/2020_12_28/components/verbale.tex

```diff
@@ -0,0 +1,96 @@
+	\item\textbf{Discussione sul linguaggio di programmazione meglio portato per la realizzazione del progetto;}
+	\item\textbf{Progettazione dell'algoritmo di sincronizzazione: modalità da preferire ed approccio corretto.}
+
+
+\end{itemize}
+
+\newpage
+
+
+\section{Resoconto}
+La riunione è stata richiesta dal gruppo \textit{\Gruppo{}} per approfondire la conoscenza del referente dell'azienda proponente e per chiarire dubbi sorti dopo una attenta analisi del capitolato.
+Durante l'incontro è stato possibile sottoporre una serie di quesiti, sia sulle funzionalità desiderate dall'azienda e sia su aspetti prettamente tecnici.
+Di seguito è riportata la lista degli argomenti affrontati.
+\subsection{Chiarimenti sulle funzionalità aggiuntive ed opzionali}
+Il proponente ha esposto le caratteristiche fondamentali del prodotto da sviluppare, evidenziando l'importanza che riveste la corretta realizzazione dell'algoritmo di sincronizzazione.
+I requisiti fondamentali sono:
+\begin{itemize}
+	\item Sviluppo dell'algoritmo di sincronizzazione;
+	\item Sviluppo di una interfaccia multipiattaforma;
+	\item Integrazione con il prodotto Zextras Drive.
+\end{itemize}
+Sono inoltre stati discussi i requisiti opzionali.
+\subsection{Discussione sulle API offerte da Zextras Drive}
+L'azienda ha esposto le API offerte dal prodotto Zextras Drive e discusso la loro documentazione con il gruppo. è stato inoltre chiarito che l'azienda potrà aggiungere API se  desiderate dal gruppo e se di possibile realizzazione.
+\subsection{Discussione sui linguaggio di programmazione utilizzabili}
+Il gruppo ha esposto e motivato una lista di possibili linguaggi scelti per la realizzazione del progetto. Essi sono:
+\begin{itemize}
+	\item Java;
+	\item C++;
+	\item Python.
+\end{itemize}
+Dopo una breve discussione con \textit{Federico} si è potuto scegliere all'unanimità Python per la semplicità ed il supporto che le diverse librerie offrono. Tutte queste qualità risultano invece mancanti negli altri linguaggi: C++ risulta complesso e scarno di librerie mentre Java è stato fortemente sconsigliato dall'azienda data la difficile installazione di un prodotto avente la JVM.
+\subsection{Requisiti dell'algoritmo di sincronizzazione}
+\textit{Alessio Crestani} ha provveduto a chiarire l'importanza che riveste la progettazione dell'algoritmo di sincronizzazione. Esso deve realizzare le scelte migliori per l'utente e permettere la perdita del minor numero di dati nei casi limite.
+
+
+\newpage
+
+\section{Riepilogo delle decisioni \hfil}
+{
+	\setlength{\freewidth}{\dimexpr\textwidth-4\tabcolsep}
+	\renewcommand{\arraystretch}{1.5}
+	\setlength{\aboverulesep}{0pt}
+	\setlength{\belowrulesep}{0pt}
+	\rowcolors{2}{AzzurroGruppo!10}{white}
+	\begin{longtable}{L{.3\freewidth} L{.7\freewidth}}
+		\toprule 
+		\rowcolor{AzzurroGruppo!30}
+		\textbf{Codice} & \textbf{Decisione}\\
+		\toprule
+		\endhead
+		
+		VE\_\DataMeeting{}.1 & Elencate le funzionalità obbligatorie ed opzionali. \\  
```

Comment:
Grammaticalmente parlando, "ed" andrebbe usato solo nei casi in cui la parola successiva inizi con la e. Se però è stato usato in altri documenti non ha senso cambiarlo qua e lasciarlo negli altri.